bot/internal/bot/bot.go

```diff
@@ -124,7 +124,8 @@ func classifyChanges(e *env.Environment, files []github.PullRequestFile) (docs b
 	switch e.Repository {
 	case env.TeleportRepo:
 		for _, file := range files {
-			if strings.HasPrefix(file.Name, "docs/") {
+			if strings.HasPrefix(file.Name, "docs/") ||
+				file.Name == "CHANGELOG.md" {
```

Comment:
What if we just check all .md files? E.g. something like
```suggestion
				strings.HasSuffix(file.Name, ".md") {
```

Reply:
I think that depends on if we want to bring RFDs under the purview of the docs team in addition to other Markdown docs in one of the repos that uses the shared workflow bot. The docs team already edits `docs/` files and `CHANGELOG.md` regularly, but we don't tend to work on other Markdown files.